src/execution/messages.cc

```diff
@@ -438,7 +438,14 @@ MaybeHandle<Object> ErrorUtils::FormatStackTrace(Isolate* isolate,
     }
   }
 
-  return builder.Finish();
+  MaybeHandle<String> rv = builder.Finish();
+  if (recordreplay::IsRecordingOrReplaying("ErrorUtils::FormatStackTrace")) {
+    // [PRO-1150] Replay error stack.
```

Comment:
NIT: We can be more concise in the comment here "Replay `Error.stack`", since that would be the actual builtin's name.